gui/wxpython/location_wizard/wizard.py

```diff
@@ -171,14 +162,26 @@ def __init__(self, wizard, parent, grassdatabase):
         self.locTitle = ''
 
         # text controls
-        self.tgisdbase = self.MakeStaticText(grassdatabase, size=(-1, -1))
-        self.tlocation = self.MakeTextCtrl("newLocation", size=(300, -1))
+        self.tgisdbase = self.MakeTextCtrl(grassdatabase, size=(400, -1), style = wx.TE_READONLY | wx.TRANSPARENT_WINDOW)       
```

Comment:
If you look at the code checks, the flake8 code quality check is failing on your PR because of spaces around =. It should be:
`..., style=wx.TE_READONLY | wx.TRANSPARENT_WINDOW`

see [PEP8](https://www.python.org/dev/peps/pep-0008/) for reference.